tests/hpc/slurm_slave.pm

```diff
@@ -13,16 +13,24 @@ use testapi;
 use serial_terminal 'select_serial_terminal';
 use lockapi;
 use utils;
+use version_utils 'is_sle';
 use hpc::utils 'get_slurm_version';
 
 sub run ($self) {
     select_serial_terminal();
     $self->prepare_user_and_group();
     my $slurm_pkg = get_slurm_version(get_var('SLURM_VERSION', ''));
 
-    # Install slurm
+    # If one wants to test unversioned slurm, one should not
+    # install slurm-node at all.
+    if ($slurm_pkg eq 'slurm' and is_sle('=12-sp5')) {
+        zypper_call("in slurm");
+    } else {
+        zypper_call("in $slurm_pkg-node $slurm_pkg-munge");
+    }
+
     # $slurm_pkg-munge is installed explicitly since slurm_23_02
-    zypper_call("in $slurm_pkg-node $slurm_pkg-munge");
+    zypper_call("in $slurm_pkg-munge");
```

Comment:
Line 33 installs `$slurm_pkg-munge`  all the time, but it is also in condition line 24 installed on line 29. 

`zypper_call("in $slurm_pkg-node $slurm_pkg-munge");`